src/main/java/com/google/api/codegen/java/JavaGapicContext.java

```diff
@@ -332,4 +332,34 @@ public String getMultilineHeading(String heading) {
   public JavaDocConfig.Builder newJavaDocConfigBuilder() {
     return JavaDocConfig.newBuilder();
   }
+
+  /**
+   * Get a Java formatted primitive value, given a primitive type and a string representation of
+   * that value. The value must be a valid example of that type. Values of type String and Bytes
+   * must contain quote characters. Values of type Bool must be either the string 'true' or 'false'
```

Comment:
regarding String/Bytes and quote characters: I see what you're doing here - piping the quote characters in from the input config. I don't think that generalizes to other languages though; e.g. for Python, we might want to use single quotes. I think you should probably strip out the quotes from the config, then add them back on for Java.